docs/topics/what-is-the-toolkit.adoc

```diff
@@ -22,7 +22,11 @@ The {ProductName} ({ProductShortName}) is an extensible and customizable rule-ba
 * Migrating from Oracle WebLogic or IBM WebSphere Application Server to Red Hat JBoss Enterprise Application Platform
 * Containerizing applications and making them cloud-ready
 * Migrating from Java Spring Boot to Quarkus
-* Updating from Oracle JDK to OpenJDK
+* Upgrading from OpenJDK 8 to OpenJDK 11
+* Upgrading from OpenJDK11 to OpenJDK 17

```

Comment:
@RichardHoch Missing space between "OpenJDK" and "11"

Reply:
Fixed.